executor/write.go

```diff
@@ -806,6 +808,12 @@ func (e *LoadData) Open(goCtx goctx.Context) error {
 	return nil
 }
 
+type defaultVal struct {
+	val types.Datum
+	// valid indicates val is a casted default value.
```

Comment:
We evaluate the default value lazily. The `valid` indicates whether the `val` is evaluated.